rrfs-test/CMakeLists.txt

```diff
@@ -81,13 +81,15 @@ if (CLONE_RRFSDATA)
              rrfs_mpasjedi_2022052619_Ens3Dvar
              rrfs_mpasjedi_2022052619_letkf
              rrfs_mpasjedi_2022052619_atms_npp_qc
+             rrfs_mpasjedi_2022052619_atms_npp_qc_bc
           )
 
       # JEDI executables for each test case above
       list ( APPEND rrfs_mpasjedi_exes
              mpasjedi_variational.x
              mpasjedi_enkf.x
              mpasjedi_variational.x
+             mpasjedi_variational.x
```

Comment:
We need a more compact way to deal with this "executable assigning" in the future.

Reply:
That is fine!